### DISCUSSION TITLE
v3.37.2

### DISCUSSION THREAD

**Discussion by mehalter** [AstroNvim/AstroNvim#2323] [Announcements]
### Bug Fixes

* **autocmds:** don't disable `statuscolumn` for terminals in neovim >=0.9.3 ([1ed628a](https://github.com/AstroNvim/AstroNvim/commit/1ed628a7b7e0a6c597922a58af8c3f90565ef81e))

**Full Changelog**: https://github.com/AstroNvim/AstroNvim/compare/v3.37.1...v3.37.2

<hr /><em>This discussion was created from the release <a href='https://github.com/AstroNvim/AstroNvim/releases/tag/v3.37.2'>v3.37.2</a>.</em>

**Comment by matyle** [comment]
<img width="675" alt="image" src="https://github.com/AstroNvim/AstroNvim/assets/27877040/cd754e9b-d50d-4425-9c5a-f249c2b44f9e">

How do I display the relevant information on the right

**Reply by mehalter** [reply]
`<leader>lS` powered by Aerial
